Zend/zend_API.c

```diff
@@ -234,6 +234,12 @@ ZEND_API ZEND_COLD void ZEND_FASTCALL zend_wrong_parameter_error(int error_code,
 		case ZPP_ERROR_WRONG_STRING_OR_CLASS_OR_NULL:
 			zend_wrong_parameter_string_or_class_or_null_error(num, name, arg);
 			break;
+		case ZPP_ERROR_WRONG_CLASS_NAME_OR_CLASS:
+			zend_wrong_parameter_class_name_or_class_error(num, name, arg);
```

Comment:
I'm not sure that the related ZPP macros are needed anymore. I can remove them if not.

Reply:
I've just removed them from the first commit.